src/spec-node/imageMetadata.ts

```diff
@@ -299,7 +300,7 @@ export function getDevcontainerMetadata(baseImageMetadata: SubstitutedConfig<Ima
 	const raw = [
 		...baseImageMetadata.raw,
 		...featureRaw,
-		pick(devContainerConfig.raw, pickConfigProperties),
+		pick(devContainerConfig.raw, effectivePickDevcontainerProperties),
```

Comment:
Also use `effectivePickDevcontainerProperties` below for `config` for consistency. (It doesn't change the behavior in the cases we are looking at.)

Reply:
Changes made and the behavior seems the same :+1:
With the change, the method will omit `remoteEnv` in both `raw` and `metadata` properties in the response
Unit test has been updated [here (line 261)](https://github.com/devcontainers/cli/pull/453/files/bc78df5bed179fcbba1136933dddaffdf5a3c1b3..93c235bf636e37468094ad75aa203e4aab12edfd#diff-50eea78b7e48891b271d364f69b5c80f216c1cccf1d77affa79499af4d5bc1d4)